src/views/datatiles/data-tiles-directive.js

```diff
@@ -0,0 +1,349 @@
+          $scope.eventText = item.name + ' clicked\n' + $scope.eventText;
+        };
+        var handleDblClick = function (item, e) {
+          $scope.eventText = item.name + ' double clicked\n' + $scope.eventText;
+        };
+        var handleCheckBoxChange = function (item, selected, e) {
+          $scope.eventText = item.name + ' checked: ' + item.selected + '\n' + $scope.eventText;
+        };
+
+        var checkDisabledItem = function(item) {
+          return $scope.showDisabled && (item.name === "John Smith");
+        };
+
+        $scope.selectType = 'checkbox';
+        $scope.updateSelectionType = function() {
+          if ($scope.selectType === 'checkbox') {
+            $scope.config.selectItems = false;
+            $scope.config.showSelectBox = true;
+          } else if ($scope.selectType === 'tile') {
+            $scope.config.selectItems = true;
+            $scope.config.showSelectBox = false;
+          } else {
+            $scope.config.selectItems = false
+            $scope.config.showSelectBox = false;
+          }
+        };
+
+        $scope.showDisabled = false;
+
+        $scope.config = {
+         width: 240,
+         height: 75,
+         selectItems: false,
+         multiSelect: false,
+         dblClick: false,
+         selectionMatchProp: 'name',
+         selectedItems: [],
+         checkDisabled: checkDisabledItem,
+         showSelectBox: true,
+         onSelect: handleSelect,
+         onSelectionChange: handleSelectionChange,
+         onCheckBoxChange: handleCheckBoxChange,
+         onClick: handleClick,
+         onDblClick: handleDblClick
+        };
+
+        $scope.items = [
+          {
+            name: "Fred Flintstone",
+            address: "20 Dinosaur Way",
+            city: "Bedrock",
+            state: "Washingstone"
+          },
+          {
+            name: "John Smith",
+            address: "415 East Main Street",
+            city: "Norfolk",
+            state: "Virginia"
+          },
+          {
+            name: "Frank Livingston",
+            address: "234 Elm Street",
+            city: "Pittsburgh",
+            state: "Pennsylvania"
+          },
+          {
+            name: "Judy Green",
+            address: "2 Apple Boulevard",
+            city: "Cincinatti",
+            state: "Ohio"
+          },
+          {
+            name: "Pat Thomas",
+            address: "50 Second Street",
+            city: "New York",
+            state: "New York"
+          },
+        ]
+      }
+ ]);
+ </file>
+ </example>
+ */
+angular.module('patternfly.views').directive('pfDataTiles', [
+  function () {
+    'use strict';
+    return {
+      restrict: 'A',
+      scope: {
+        config: '=?',
+        items: '=',
+        eventId: '@id'
+      },
+      transclude: true,
+      templateUrl: 'views/datatiles/data-tiles.html',
+      controller: function ($scope) {
+        $scope.defaultConfig = {
+          width: '260',
+          height: '290',
```

Comment:
Why are we not just setting this in angular-pf CSS and the products can override if necessary?